packages/modal/src/component.tsx

```diff
@@ -32,9 +33,12 @@ export const Modal = ({
   }, [props.open, contentRef]);
 
   useEffect(() => {
-    if (!props.initialFocusRef) return;
-    props.initialFocusRef.current?.focus();
-  }, [props.open, props.initialFocusRef]);
+    if (!props.initialFocusRef) {
+      props.right && closeButtonRef.current?.focus();
+    } else {
+      props.initialFocusRef.current?.focus();
+    }
+  }, [props.open, props.initialFocusRef, closeButtonRef]);
```

Comment:
Do we need `closeButtonRef` in the dependency array? If so, should we add `props.right`, too?

Reply:
ah, no you are right, that can be removed. And the `props.right` should be added instead. Don't think it is really necessary though but it can change and re-render the component

Reply:
https://github.com/warp-ds/react/pull/128/commits/54f4c5f48a1da68a12d584bbc840a6dc799aa101